Gemfile.lock

```diff
@@ -424,6 +424,5 @@ DEPENDENCIES
   therubyracer
   uglifier (~> 2.0.1)
   wraith (~> 1.3.0)
-
```

Comment:
You'll need to revert this. It will cause issues with the docker PR builds and the live build